src/server/cmd/pachctl/cmd/cmd.go

```diff
@@ -345,9 +343,10 @@ kubectl %v port-forward "$pod" %d:8080
 			eg.Go(func() error {
 				fmt.Printf("Forwarding the dash (Pachyderm dashboard) UI port to http://localhost:%v ...\n", uiPort)
 				stdin := strings.NewReader(fmt.Sprintf(`
-pod=$(kubectl %v get pod -l app=dash | awk '{if (NR!=1) { print $1; exit 0 }}')
+pod=$(kubectl %v get pod -l app=dash --output='jsonpath={.items[0].metadata.name}')
 kubectl %v port-forward "$pod" %d:8081
 `, kubeCtlFlags, kubeCtlFlags, uiWebsocketPort))
+				fmt.Println(stdin)
```

Comment:
Is this debug output? Or something we want the user to see?

Reply:
Yup, nice catch.